core/scripts/vrfv2plus/testnet/super_scripts.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"math/big"
+	"os"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/shopspring/decimal"
+
+	helpers "github.com/smartcontractkit/chainlink/core/scripts/common"
+	"github.com/smartcontractkit/chainlink/v2/core/assets"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/link_token_interface"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrf_coordinator_v2plus"
+	"github.com/smartcontractkit/chainlink/v2/core/services/signatures/secp256k1"
+)
+
+const formattedVRFJob = `
+type = "vrf"
+name = "vrf_v2_plus"
+schemaVersion = 1
+coordinatorAddress = "%s"
+batchCoordinatorAddress = "%s"
+batchFulfillmentEnabled = %t
+batchFulfillmentGasMultiplier = 1.1
+publicKey = "%s"
+minIncomingConfirmations = 3
+evmChainID = "%d"
+fromAddresses = ["%s"]
+pollPeriod = "5s"
+requestTimeout = "24h"
+observationSource = """
+decode_log              [type=ethabidecodelog
+                         abi="RandomWordsRequested(bytes32 indexed keyHash,uint256 requestId,uint256 preSeed,uint64 indexed subId,uint16 minimumRequestConfirmations,uint32 callbackGasLimit,uint32 numWords,bool nativePayment,address indexed sender)"
+                         data="$(jobRun.logData)"
+                         topics="$(jobRun.logTopics)"]
+generate_proof          [type=vrfv2plus
+                         publicKey="$(jobSpec.publicKey)"
+                         requestBlockHash="$(jobRun.logBlockHash)"
+                         requestBlockNumber="$(jobRun.logBlockNumber)"
+                         topics="$(jobRun.logTopics)"]
+estimate_gas            [type=estimategaslimit
+						 to="%s"
+						 multiplier="1.1"
+						 data="$(generate_proof.output)"]
+simulate_fulfillment    [type=ethcall
+                         to="%s"
+		                 gas="$(estimate_gas)"
+		                 gasPrice="$(jobSpec.maxGasPrice)"
+		                 extractRevertReason=true
+		                 contract="%s"
+		                 data="$(generate_proof.output)"]
+decode_log->generate_proof->estimate_gas->simulate_fulfillment
+"""
+`
+
+func deployUniverse(e helpers.Environment) {
+	deployCmd := flag.NewFlagSet("deploy-universe", flag.ExitOnError)
+
+	// required flags
+	linkAddress := deployCmd.String("link-address", "", "address of link token")
+	linkEthAddress := deployCmd.String("link-eth-feed", "", "address of link eth feed")
+	subscriptionBalanceString := deployCmd.String("subscription-balance", "1e19", "amount to fund subscription")
```

Comment:
we could add capability to add eth balance to this script. not a blocking comment though

Reply:
same goes for wrapper

Reply:
Sure, I'll add these a changes as a follow up PR.